examples/gallery/embellishments/timestamp.py

```diff
@@ -0,0 +1,21 @@
+"""
+Timestamp
+---------
+The :meth:`pygmt.Figure.timestamp` method can draw the
+GMT timestamp logo on the map/plot. A custom label
+can be added via the ``label`` parameter.
+"""
+
+import pygmt
+
+# Define static format string.
+pygmt.config(FORMAT_TIME_STAMP="2023-03-01T20:45:15")
```

Comment:
We don't need to have a static UNIX timestamp string here, because the gallery examples are not tested in the CI.